style.css

```diff
@@ -110,3 +138,238 @@ main {
   margin: 10px;
   list-style: none;
 }
+
+.snapshoot {
+  width: inherit;
+  height: 220px;
+  display: block;
+}
+
+.snapshoot > img {
+  width: 100%;
+  height: 100%;
+}
+
+.frame {
+  width: 236px;
+  height: 32px;
+  display: grid;
+  margin: 10px 0;
+  align-items: center;
+  grid-template-columns: auto auto auto auto auto;
+}
+
+.frame > p,
+.frame > h2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.frame > h2 {
+  color: #344563;
+}
+
+.frame > div {
+  background-color: #e5e5e5;
+  border-radius: 50%;
+  height: 8px;
+  width: 8px;
+}
+
+.text > p {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
```

Comment:
- [ ] Great job styling this paragraph 👏  but please, keep in mind that this section uses its own `font-family`, you should consider using the same typography as provided in the Figma design:

![image](https://user-images.githubusercontent.com/66967274/148630079-d5c89394-b298-44d9-8eab-952d38ea1ab7.png)